doc/classes/AnimationNodeBlendSpace2D.xml

```diff
@@ -105,39 +119,49 @@
 			<argument index="1" name="pos" type="Vector2">
 			</argument>
 			<description>
+				Updates the position of the point at index [code]point[/code] on the blend axis.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="auto_triangles" type="bool" setter="set_auto_triangles" getter="get_auto_triangles">
+			If true, the blend space is triangulated automatically. The mesh updates every time you add or remove points with [method add_blend_point] and [method remove_blend_point].
 		</member>
 		<member name="blend_mode" type="int" setter="set_blend_mode" getter="get_blend_mode" enum="AnimationNodeBlendSpace2D.BlendMode">
+			Controls the interpolation between animations. See [code]BLEND_MODE_*[/code] constants.
```

Comment:
We now have proper support for `[enum]` hyperlinks, so it's better to use "See [enum BlendMode] for possible values." or similar.